### ISSUE TITLE
FP S3928: should not raise an issue in add/remove, when nameof(event_name) is used

### ISSUE THREAD

**Issue by jirikopecky** [SonarSource/sonar-dotnet#6235]
### Description

Rule should not raise an issue in add/remove parts of event, when nameof(event_name) is used.
Using nameof(value) here makes no sense, since it produces the exception text that is too generic.

```
Unhandled exception. System.ArgumentNullException: Value cannot be null. (Parameter 'value')
```

This is pretty much the same issue as https://github.com/SonarSource/sonar-dotnet/issues/4679 just for events with custom add/remove accessor implementation.

### Repro steps

```
public class EventWithCustomAccessors
{
    private event EventHandler _sampleEvent;
    public event EventHandler SampleEvent
    {
        add
        {
            if (value == null)
            {
                throw new ArgumentNullException(nameof(SampleEvent));
            }

            _sampleEvent += value;
        }

        remove
        {
            if (value == null)
            {
                throw new ArgumentNullException(nameof(SampleEvent));
            }

            _sampleEvent -= value;
        }
    }
}
```

### Expected behavior

Issue should not be raised.

### Actual behavior

Issue is raised.

### Related information

* C#/VB.NET Plugins version 8.22.0.31243 (SonarQube Enterprise Edition 8.9.6 (build 50800))
* SonarScanner for .NET version (if used) 5.8

**Comment by pavel-mikula-sonarsource** [comment]
Ahoj Jirko,

Thank you for reporting this case, I confirm it as False Positive.

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1813](https://sonarsource.atlassian.net/browse/NET-1813)

[NET-1813]: https://sonarsource.atlassian.net/browse/NET-1813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
